---
title: "PGPg_finder: A comprehensive and user-friendly pipeline for identifying plant growth-promoting genes in genomic and metagenomic data"
collection: publications
category: manuscripts
permalink: /publication/2024-05-23-paper-title-number-1
excerpt: 'Identifying and comparing plant growth-promoting traits (PGPT) within whole-genome and metagenomic
sequencing data can significantly advance agricultural research and promote sustainable crop production. This
study introduces PGPg_finder, a comprehensive pipeline designed to annotate and compare PGPT from both
whole-genome and metagenome sequencing datasets. This pipeline utilizes direct sequence annotation alongside
de novo assembly methods to accurately detect PGPT. By cross-referencing sequences from the PLaBAse data
base, it identifies and quantifies the presence of these genes within the original datasets, facilitating an intuitive
comparison of the abundance and distribution of PGPT across various samples. We evaluated the performance of
PGPg_finder by analyzing genomes from five rhizobacterial strains: Paenibacillus vini, Paenibacillus polymyxa,
Fictibacillus sp., Brevibacillus agri, and Bacillus cereus, and also metagenomic samples from bulk soils subjected to
forest-to-pasture conversion in the Amazon rainforest. The genomic workflow revealed several genes associated
with substrate utilization, abiotic stress neutralization, phosphate solubilization, and iron acquisition. It also
identified genes unique to specific lineages, including those associated with colonization and plant-derived
substrate usage in P. polymyxa, quorum sensing response and biofilm formation in P. vini, heavy metal detoxi
fication and nitrogen acquisition in B. agri, and spore production and neutralizing biotic stress in B. cereus. The
strain Fictibacillus sp. presented several unique genes related to surface attachment, stress response, xenobiotic
degradation, phosphate solubilization, and phytohormone production. The use of PGPg_finder highlights its po
tential to uncover novel inoculants and strains. The metagenomic workflow distinguished plant-growth pro
motion gene profiles between soils from the Amazon rainforest and pasture, with the latter showing a profile
more aligned with simple carbohydrate consumption, abiotic stress tolerance, motility and chemotaxis, and
phosphorus mineralization. Native forests exhibited a profile associated with the degradation of complex organic
matter, oxidative stress tolerance, xenobiotic degradation, bactericidal activity, iron acquisition, and volatile
pathways. These findings underscore the effectiveness and sensitivity of PGPg_finder in accurately identifying and
comparing PGPT genes, highlighting both commonalities and variations across samples. The application of this
pipeline has the potential to significantly facilitate the identification of plant growth-promoting microbes. '
date: 2024-05-24
venue: 'Rhizosphere'
paperurl: 'https://guto-monteiro.github.io/files/pellegrinetti et al 2024.pdf'


citation: 'Pellegrinetti, Thierry Alexandre, et al. "PGPg_finder: A comprehensive and user-friendly pipeline for identifying plant growth-promoting genes in genomic and metagenomic data." Rhizosphere 30 (2024): 100905.'
---

The contents above will be part of a list of publications, if the user clicks the link for the publication than the contents of section will be rendered as a full page, allowing you to provide more information about the paper for the reader. When publications are displayed as a single page, the contents of the above "citation" field will automatically be included below this section in a smaller font.
